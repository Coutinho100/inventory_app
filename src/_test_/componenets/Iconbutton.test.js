

import React from 'react';

import IconButton from '../../components/Component/IconButton';

import { Text} from 'react-native';

import { configure } from 'enzyme';

import Adapter from 'enzyme-adapter-react-16';

import test_function from './test_function';



configure({ adapter: new Adapter() });


test_function(["<Button view/>",0],['renders correctly','renders the length of view'],[<IconButton />,Text],null)
