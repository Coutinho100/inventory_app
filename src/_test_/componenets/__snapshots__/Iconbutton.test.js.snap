// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button view/> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "width": "100%",
    }
  }
>
  <View
    accessible={true}
    focusable={false}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "borderRadius": 10,
          "elevation": 1,
          "flex": 1,
          "height": 120,
          "margin": 15,
          "overflow": "visible",
          "width": "30%",
        }
      }
    >
      <View
        style={
          Object {
            "backgroundColor": undefined,
            "borderRadius": 10,
            "flex": 1,
            "padding": 10,
            "shadowColor": "black",
            "shadowOffset": Object {
              "height": 2,
              "width": 0,
            },
            "shadowOpacity": 0.26,
            "shadowRadius": 1,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "flex": 1,
              "height": "100%",
              "justifyContent": "center",
              "opacity": 0.7,
              "width": "100%",
            }
          }
        >
          <Text />
        </View>
      </View>
    </View>
  </View>
</View>
`;
